test/JDBC/expected/BABEL-4815-vu-verify.out

```diff
@@ -0,0 +1,268 @@
+BEGIN TRANSACTION babel_4815
+GO
+
+SELECT set_config('debug_parallel_query', '1', true)
+SELECT set_config('parallel_setup_cost', '0', true)
+SELECT set_config('parallel_tuple_cost', '0', true)
+select set_config('max_parallel_workers_per_gather', '1', true);
```

Comment:
why is_local = true being set here?

Reply:
It'll need the transaction rollback/commit to reset those parameters, since I don't want to rely on what's the original value of those parameters.